### PR TITLE
Update to use RCUE styling

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
- Add RCUE dependency
- Bumps patternfly-react to 1.6.0
- Updates nav bar top border
- Updates masthead imagery
- Updates About Modal

### CODE REVIEW

File: client/scripts/api.sh [hunk 1]
```diff
@@ -17,7 +17,7 @@ mockApi()
 
   if [ -z "$(docker ps | grep $CONTAINER)" ] && [ "$UPDATE" = false ]; then
     echo "Starting API..."
-    docker run -d --rm -p $PORT:8000 -v "$FILE:/data/swagger.yaml" --name quipucords-mock $CONTAINER >/dev/null
+    docker run -d --rm -p $PORT:8000 -v "$FILE:/data/swagger.yaml" --name quipucords-mock $CONTAINER
```
Comment (cdcabrera) [inline client/scripts/api.sh]: Probably good to remove this change from the PR

File: client/src/styles/entitlements/_overrides.scss [hunk 2]
```diff
@@ -11,7 +11,12 @@ button.unavailable:hover, li.unavailable > a:hover {
   opacity: 0;
 }
 
-.navbar-brand .navbar-brand-icon, .modal-dialog .modal-footer > img {
-  height: 35px;
-}
+.about-modal-pf {
+  background-color: #1a1a1a;
+  background-image: url('/assets/img/About-Modal-background.png');
```
Comment (cdcabrera) [inline client/src/styles/entitlements/_overrides.scss]: We including this with RCUE down the road?

There's kind of a hidden aspect of the Create-React-App with Webpack... you can make the path relative, like so
```
background-image: url('../images/About-Modal-background.png');
```

And it should run and compile with the updated absolute path.

Comment (jeff-phillips-18) [reply client/src/styles/entitlements/_overrides.scss]: I'm not sure what the RCUE plan is for this. It currently copies images from patternfly so it does not have the correct image for this.

Comment (jeff-phillips-18) [reply client/src/styles/entitlements/_overrides.scss]: Excellent, all images in one place now.

**Pull request merged**